radixdlt-cli/src/main/java/com/radixdlt/cli/GetStoredAtoms.java

```diff
@@ -33,25 +37,29 @@
  * </pre>
  */
 @CommandLine.Command(name = "get-stored-atoms", mixinStandardHelpOptions = true,
-        description = "Get stored Atoms")
-class GetStoredAtoms implements Runnable {
-    @CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
-    Composite.IdentityInfo identityInfo
-
-    @Override
-    void run() {
-        RadixApplicationAPI api = Utils.getAPI(identityInfo)
-
-        println "Retrieving atoms..."
-        api.pull()
-        def atomStore = api.getAtomStore()
-        def observations = atomStore.getAtomObservations(api.getAddress())
-        observations.filter({ it -> return it.isHead() }).blockingFirst()
-
-        println "Atom ID's:"
-        atomStore.getStoredAtoms(api.getAddress()).each(it -> printf("  %s", it.getAid()))
-        println "Done"
-        System.exit(0)
-    }
+		description = "Get stored Atoms")
+public class GetStoredAtoms implements Runnable {
+	@CommandLine.ArgGroup(exclusive = true, multiplicity = "0..1")
+	private Composite.IdentityInfo identityInfo;
+
+	@Override
+	public void run() {
+		RadixApplicationAPI api = Utils.getAPI(identityInfo);
+
+		println("Retrieving atoms...");
+		api.pull();
+		var atomStore = api.getAtomStore();
+		var observations = atomStore.getAtomObservations(api.getAddress());
+		observations.filter(AtomObservation::isHead).blockingFirst();
+
+		println("Atom ID's:");
+		atomStore.getStoredAtoms(api.getAddress()).forEach(it -> printf("  %s", it.getAid()));
```

Comment:
I think this just prints out all atom IDs in a big list on a single line.  Is this what we want, or are we missing a %n in the output?

Reply:
fixed